nc_time_axis/__init__.py

```diff
@@ -343,7 +343,10 @@ def convert(cls, value, unit, axis):
             # Don't do anything with numeric types.
             if munits.ConversionInterface.is_numlike(value):
                 return value
-            first_value = value
+            elif isinstance(value, (list, tuple)):
+                first_value = value[0]
```

Comment:
nit: I think this might be a little clearer to read if it were added as its own `elif` clause at the same level as the `np.ndarray` clause.

Reply:
We need to catch the case of a list of numbers first. I though this was the simplest way to write it, I can add comments.

Reply:
Ah I see, my mistake!  Thanks for the clarification.